style.md

```diff
@@ -375,8 +375,8 @@ func (s *Stats) Snapshot() map[string]int {
   return s.counters
 }
 
-// snapshot is no longer protected by the mutex, so any
-// access to the snapshot is racy.
+// snapshot is no longer protected by the mutex,
+// so any access to the snapshot is subject to race condition.
```

Comment:
```suggestion
// so any access to the snapshot is subject to race conditions.
```

Reply:
- I think there's no need to move `so any`
 - optionally can call out data races specifically

```
// snapshot is no longer protected by the mutex, so any
// access to the snapshot is subject to data races.
```

Reply:
I like it. I will update it very soon. thanks